### ISSUE TITLE
Fathom tracking on Docs site

### ISSUE THREAD

**Issue by intheskies24** [decentralized-identity/universal-resolver#417]
Please can we add Fathom tracking pixel the the `<head>` of the docs site pages:

Here is our embed code:
```
<!-- Fathom - beautiful, simple website analytics -->
  <script src="https://cdn.usefathom.com/script.js" data-site="OFAOQQQH" defer></script>
<!-- / Fathom -->
```

Here is a guide on how to add it to our site: https://usefathom.com/docs/script/embed

**Comment by peacekeeper** [comment]
Hmm could you summarize what is Fathom and what benefit would it give us to add it?

Also, could you create a PR that adds it?
